### PR TITLE
bump version to 5.4.2

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: Gemfile.jruby-1.9.lock [hunk 1]
```diff
@@ -201,12 +201,13 @@ GEM
     logstash-filter-fingerprint (3.0.3)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       murmurhash3
-    logstash-filter-geoip (4.0.4-java)
+    logstash-filter-geoip (4.1.1-java)
```
Comment (jordansissel) [inline Gemfile.jruby-1.9.lock]: minor bump ok here?

File: Gemfile.jruby-1.9.lock [hunk 2]
```diff
@@ -232,10 +233,8 @@ GEM
       thread_safe
     logstash-filter-urldecode (3.0.3)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-filter-useragent (3.0.3)
+    logstash-filter-useragent (3.1.1-java)
```
Comment (jordansissel) [inline Gemfile.jruby-1.9.lock]: minor bump ok here?

File: Gemfile.jruby-1.9.lock [hunk 3]
```diff
@@ -293,10 +292,10 @@ GEM
       puma (~> 2.16, >= 2.16.0)
       rack (~> 1)
       stud
-    logstash-input-http_poller (3.1.1)
+    logstash-input-http_poller (3.3.0)
```
Comment (jordansissel) [inline Gemfile.jruby-1.9.lock]: minor bump ok here?

File: Gemfile.jruby-1.9.lock [hunk 4]
```diff
@@ -421,9 +422,9 @@ GEM
       logstash-core-plugin-api (>= 2.0.0, < 2.99)
     logstash-output-graphite (3.1.1)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
-    logstash-output-http (4.2.0)
+    logstash-output-http (4.3.0)
```
Comment (jordansissel) [inline Gemfile.jruby-1.9.lock]: minor bump ok here?

**Comment by jordansissel** [comment]
LGTM other than my comments about plugin minor bumps in this logstash patch bump

**Comment by jordansissel** [comment]
3/4 travis jobs succeeded. THe one failure seems unrelated.

**Comment by suyograo** [comment]
@jordansissel those changes should be ok. We've been including plugin minors in point releases. As long as they arent breaking changes, we should be fine.

**Pull request merged**